packages/ilios-common/addon/components/dashboard/courses-calendar-filter.js

```diff
@@ -5,7 +5,6 @@ import { action } from '@ember/object';
 import { service } from '@ember/service';
 import { DateTime } from 'luxon';
 import { findBy, sortBy } from 'ilios-common/utils/array-helpers';
-import scrollIntoView from 'scroll-into-view';
```

Comment:
So, no more scrolling at all?

Reply:
Not using this helper anymore, just doing the scroll using the [scrollTo](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo) API on the element.